doc/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+project(doc CXX)
+cmake_minimum_required(VERSION 3.3.0)
+
+find_program(POD2HTML_PATH pod2html)

```

Comment:
Something like `if (PODT2HTML_PATH) ..... else () message(FATAL_ERROR "pod2html not found") endif ()` might be a good idea since the command won't fail if it's missing.

Reply:
done